views/booking.njk

```diff
@@ -113,15 +114,15 @@
             {% endif %}
 
             <input 
-                class="govuk-input lbh-input govuk-input--width-20" 
+                class="govuk-input lbh-input {{ 'govuk-input--error' if errorFields.email }} govuk-input--width-20" 
                 id="booking_email" 
                 name="booking_email" 
                 type="text" 
                 value="{{ oldInput.email }}">
         </div>
 
         <div class="govuk-form-group lbh-form-group">
-            <label class="govuk-label  lbh-label" for="input-with-hint-text">
+            <label class="govuk-label lbh-label" for="input-with-hint-text">
                 Contact number (optional)
             </label>
 
```

Comment:
Should the contact number input be of type "tel"?

Reply:
Good point. I'll make the change thanks.